apps/web/src/routes/_content/_misc/public-communication/blogs-and-news/$category/$name.tsx

```diff
@@ -79,12 +79,13 @@ function SingleBlogDetail() {
                 {blog.tags && (
                   <div className="flex flex-row">
                     {blog.tags.map((tag, index) => (
-                      <span
+                      <TextTag
```

Comment:
Add gap between the tags (in the parent div)

Reply:
![image](https://github.com/user-attachments/assets/d52746bb-0670-4355-8b7a-0b8695e76393)